src/containers/BooksForm.js

```diff
@@ -10,25 +12,57 @@ const categories = [
   'Sci-Fi',
 ];
 
-const BooksForm = () => (
-  <form>
-    <h4> Title:</h4>
-    <input
-      type="text"
-      name="title"
-      value="{title}"
-    />
-    <h4>Category:</h4>
-    <select
-      name="category"
-      value=""
-    >
-      {categories.map((category) => (
-        <option key={category}>{category}</option>
-      ))}
-    </select>
-    <button type="submit">Submit</button>
-  </form>
-);
+const BooksForm = () => {
+  const [book, setBook] = useState({ title: '', category: '' });
+  // const books = useSelector((store) => store.books);
```

Comment:
### Optional
- [ ] It would be better if we remove commented piece of code. It makes the codebase messy

Reply:
Thanks for the heads up, done.